lib/queuePopulator/IngestionProducer.js

```diff
@@ -166,7 +166,9 @@ class IngestionProducer {
                         if (err) {
                             return next(err);
                         }
-                        initialCseq = cseq;
+                        // we should perform snapshot phase up to this cseq, so
+                        // save bookmark at cseq + 1
+                        initialCseq = cseq + 1;
```

Comment:
This should be a commit of it's own as it's changing the behavior of QueuePopulator/IngestionProducer.